src/nccl_ofi_rdma.c

```diff
@@ -3229,6 +3237,14 @@ static int recv_close(nccl_net_ofi_recv_comm_t *recv_comm)
 		goto exit;
 	}
 
+	/* Destroy domain */
+#if HAVE_NVTX_TRACING
+	if (NCCL_OFI_NVTX_TRACE_PER_COMM)
```

Comment:
why not prefer `#if HAVE_NVTX_TRACING && NCCL_OFI_NVTX_TRACE_PER_COMM`?

Reply:
Done